R_program/Vegetation.R

```diff
@@ -0,0 +1,116 @@
+#The Burke-Lauenroth Laboratory
+#STEPWAT R Wrapper
+# Vegetation code for STEPWAT Wrapper
+
+
+#' Function to estimate relative abundance of functional groups based on
+#' climate relationships
+#'
+#' @param sw_weatherList A list. An object as created by the function
+#'   \code{\link{extract_data}} of the script
+#'   \var{"RSoilWat31.Weather.Data.Query_V2.R"}. It is a list with an element
```

Comment:
update the file name here - this file no longer exists

Reply:
@kpalmqui Do you know where this functionality exists now? we might have to ask Daniel what this file was renamed to.

Reply:
@chaukap this is not an rSOILWAT2 file, rather a rSFSTEP2 file. It is now called WeatherQuery.R

Reply:
Resolved by commit f273812.